mosquito/mapi.py

```diff
@@ -7,7 +7,8 @@
 import time
 import mosquito.msppg as msppg
 from mosquito.coms import MosquitoComms
-
+from mosquito.pid_parameters import flightModeController as controller
```

Comment:
i don't see where this import is used Pep. Maybe there is a missing file in the PR or the imports are not used?